swap_meet/clothing.py

```diff
@@ -1,2 +1,11 @@
-class Clothing:
-    pass
+from .item import Item
+
+
+class Clothing(Item):
+    def __init__(self, condition=0):
+        super().__init__(condition)
+        self.category = "Clothing"
+        self.condition = condition
```

Comment:
You updated `Item`'s constructor but not how you called it:

```suggestion
        super().__init__("Clothing", condition)
```